std_msgs/Int8 systemMode
std_msgs/Int8 systemSignal
std_msgs/Int8 systemState
std_msgs/Int8 systemHealth